google/cloud/aiplatform/training_jobs.py

```diff
@@ -4172,6 +4172,202 @@ def run(
             sync=sync,
         )
 
+    def _run_with_experiments(
```

Comment:
It seems like this was reverted back in.

Reply:
I think it was the merges.
I've re-done the merges.